rasa/core/constants.py

```diff
@@ -26,3 +26,11 @@
 ACTION_NAME_SENDER_ID_CONNECTOR_STR = "__sender_id:"
 
 BEARER_TOKEN_PREFIX = "Bearer "
+
+DEFAULT_POLICY_PRIORITIES = {
+    "Default": 1,
```

Comment:
can we use the class_name as key?

Reply:
probably better do sth like `"TwoStageFallbackPolicy"` than `TwoStageFallbackPolicy.__name__` to avoid imports. But the classes themselves should access the policies with sth like `DEFAULT_POLICY_PRIORITIES[self.__class__.__name__]` (I'm not entirely sure about the exact code)

Reply:
I thought about it but all ML policies use default value `1`, should I add all of them there? Also `FallbackPolicy` and `TwoStageFallbackPolicy` use the same priority

Reply:
And if users create custom policies, it seems to me, this keys are easier to use

Reply:
or we do sth like `priority: int = POLICY_PRIORITIES.get(self.__class__.__name__, DEFAULT_PRIORITY)` ?

> And if users create custom policies, it seems to me, this keys are easier to use

 Why are these easier to use?

Reply:
@tabergma what do you think?

Reply:
reminder: Use a `defaultdict`

Reply:
use constants :)

Reply:
used constants :)